Observer/Login.php

```diff
@@ -8,24 +8,27 @@
 
 class Login implements ObserverInterface
 {
-   
+    protected $_customerFactory;
+    protected $_testFactory;
      public function __construct(LoggerInterface $logger,
-            \Excellence\Event\Model\TestFactory $testFactory )
+            \Excellence\Event\Model\TestFactory $testFactory,
+            \Magento\Customer\Model\CustomerFactory $customerFactory )
     {
         $this->logger = $logger;
         $this->_testFactory = $testFactory;
+        $this->_customerFactory=$customerFactory;
     }
  
     public function execute(Observer $observer)
     {   
           $test=$this->_testFactory->create();
+          $customer=$this->_customerFactory->create();
           $event = $observer->getEvent();
-          $id = $event->getCustomer()->getId();  
-          $objectManager = \Magento\Framework\App\ObjectManager::getInstance();
-          $customer_data = $objectManager->create('Magento\Customer\Model\Customer')->load($id);
-          $customer_email = $customer_data->getEmail();
-          $time=(new \DateTime())->format(\Magento\Framework\Stdlib\DateTime::DATETIME_PHP_FORMAT);
-          $test->saveData($id,$customer_email,$time);
+          $id = $event->getCustomer()->getId();
+          $customerEmail = $customer->load($id);
```

Comment:
is it required to load customer? 
customer object should be already coming loaded with data